### ISSUE TITLE
Cannot import certain video files.

### ISSUE THREAD

**Issue by SomeRandomDude67** [OpenShot/openshot-qt#4544]
Description:
I was trying to import some videos I had recorded via OBS. I mainly use the same settings every time and never change them, and I was able to import previous files before with the same settings. However, after recording 5 clips I wanted to add them all together, so I opened up OpenShot and imported those 5 files by dragging them in. Unfortunately, it crashed, and I had no idea as to how, or why. I tried it again several times dragging them in, and selecting them, even doing them 1 at a time to no avail. I then thought, maybe I couldn't import anything at all. Nope. I went to some of the videos I downloaded from YouTube and they all could be imported with no issue. Then I thought maybe it's the issue with OBS so I brought back one of my old recorded videos from OBS for my school work and it imported with ease. Then I thought, maybe it's the recently recorded videos instead. So I recorded a small video that has nothing to do with what I wanted to record, and it imported with no issues as well. Further investigation and I found out that it's only those 5 videos that have issues importing. I'd gladly be able to re-record them again but I would like some info if possible as to why those 5 videos are the only ones with problems.

Yeah... I don't know how to reproduce this issue sorry :(

Expected behavior:
I was expecting it to import those 5 videos.

System Details:
- OpenShot 2.6.1
- Windows 10

Log Files:

[openshot-qt.log](https://github.com/OpenShot/openshot-qt/files/7569658/openshot-qt.log)

Exception / Stacktrace:

idk what this is

**Comment by JacksonRG** [comment]
Are any of the videos ones you're willing to share (or share a few seconds of one).

**As far as solving your immediate problem:**
I'd suggest downloading [handbrake](https://handbrake.fr/) and perhaps re-encoding your files into a different format. I'd love to hear if that works. And if it does, you won't have to re-record.

Glad to hear it's not persistent with your OBS videos. But if you trim one to just a few seconds (handbrake should help with this too) we may be able to see why it's crashing.

**Comment by SomeRandomDude67** [comment]
They're not that long actually, around 20 seconds or so of me just showcasing some stuff from a game.


https://user-images.githubusercontent.com/71267236/142650759-9279db41-2774-4614-8793-e4660bd8a15a.mp4

https://user-images.githubusercontent.com/71267236/142650777-0e6a3782-044b-4cc5-a8c8-6accb9e99a3c.mp4

https://user-images.githubusercontent.com/71267236/142650788-57f75e43-2833-4b0b-a7dd-1a3f8a3e7855.mp4

https://user-images.githubusercontent.com/71267236/142650798-4f73d66b-7293-4b8f-a32f-da09927954f6.mp4

https://user-images.githubusercontent.com/71267236/142650814-d91521a2-a058-4654-b014-f1db7ce96381.mp4


I also tried that handbrake thing you mentioned but alas, it too, could not work as well.

**Comment by JacksonRG** [comment]
Thanks! I'll take a look

**Comment by JacksonRG** [comment]
Just tested, and it imported no problem.

Gonna look at the log files. But I'd recommend deleting your `C:\Users\userName\.openshot_qt\` folder, and reinstalling Openshot in the mean time.

**Comment by mpincai** [comment]
Hi, I am having a similar issue. 

Several people have sent me a video to compile and out of 47 items (saved as .MOV and .MP4 extensions), I am having issues with 3 of the videos. I have been able to upload all other 44 videos with no issues but these 3 always crash OpenShot without any error message.

I already tried uninstalling and installing the app again. And also tried using handbrake with no success. I still have the same issue. 

Since this is a video recorded by someone else, I would really really prefer to have this issue corrected rather than having these recorded again. 

I am attaching the logfile for reference.
[openshot-qt.log](https://github.com/OpenShot/openshot-qt/files/7622288/openshot-qt.log)
